inst/teal_slices.yml

```diff
@@ -0,0 +1,54 @@
+schemas:
+  teal_slices:
+    type: object
+    required: # this part of the response is always returned, attributes object is optional
+      - slices
+    properties:
+      slices:
+        type: array
+        items:
+          type: object
+          required: # those 3 fields below are always returned, the other are optional in the response
+            - dataname
+            - id
+            - locked
+          properties:
+            dataname:
+              type: string
+            varname:
+              type: string
+            choices:
+              type: array
+              items:
+                type: string
```

Comment:
`choices` may also be numeric, logical, date and date-time, see `teal.slice:::.filterable_class`
it can also be NULL

Reply:
thanks - will extend the types